storage.go

```diff
@@ -734,18 +646,15 @@ func (s *Storage) write(ctx context.Context, path string, r io.Reader, size int6
 	} else {
 		r = io.LimitReader(r, size)
 	}
-
 	if opt.HasIoCallback {
 		r = iowrap.CallbackReader(r, opt.IoCallback)
 	}
-
 	input, err := s.formatPutObjectInput(path, size, opt)
 	if err != nil {
 		return
 	}
-
-	input.Body = aws.ReadSeekCloser(r)
-	_, err = s.service.PutObjectWithContext(ctx, input)
+	input.Body = manager.ReadSeekCloser(r)
```

Comment:
`input.Body` requires a `io.Reader` now, maybe we don't need to use `manager.ReadSeekCloser(r)` any more.

Reply:
Yes. manager.ReadSeekCloser(r) returns a io.Reader indirectly.We can use io.Reader directly.